File: portfolio-new/web/scripts/extract_cv.js
const fs = require('fs');
const pdfParse = require('pdf-parse');
const pdf = pdfParse.default || pdfParse;

let dataBuffer = fs.readFileSync('/Users/debarghabandyopadhyay/Dev/spore-portfolio/portfolio-new/Debargha_CV.pdf');

pdf(dataBuffer).then(function(data) {
    console.log(data.text);
}).catch(function(err) {
    console.error(err);
});

File: portfolio-new/web/scripts/extract_cv.py
from pypdf import PdfReader

reader = PdfReader("/Users/debarghabandyopadhyay/Dev/spore-portfolio/portfolio-new/Debargha_CV.pdf")
text = ""
for page in reader.pages:
    text += page.extract_text() + "\n"

with open("extracted_cv.txt", "w", encoding="utf-8") as f:
    f.write(text)
print("CV extracted successfully.")
